frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -0,0 +1,141 @@
+
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import * as fuzzy from 'fuzzysearch';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import * as PropTypes from 'prop-types';
+import { Firehose, LoadingInline, Dropdown, ResourceName} from '../utils';
+
+/* Component ListDropDown - creates a dropdown list of resources specified */
+
+class ListDropdown_ extends React.Component {
```

Comment:
Is this based on ListDropdown from the binding page? If so, we should try to remove that one and use the common component

Reply:
I was going to open a new PR to replace the code on the binding page.  Just trying to reduce the scope of this PR for now.  I opened CONSOLE-896 to track this.